Additional rewriting (#2)
jbee/jbee.github.io: purejin/index.md

* use your list/set implementations instead of javas (is really easy)
* binds different configurations without `if`s everywhere (built-in concepts)
* binds in different scopes

> * uses your own scopes/lifecycles (is really easy)

Reviewer: ```suggestion
* can use your own scopes/lifecycles (easy to setup)
```

> * uses your own provider interface to escape scoping problems (is really easy) 

Reviewer: This is more confusing now, it does not use _your own_ out of the box but it is easy to setup _your own_.

Author: Updated. What do you think?

Reviewer: Better, Thanks. I apply it to the scopes item as well.
